app/src/main/res/layout/main.xml

```diff
@@ -0,0 +1,29 @@
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
Is this class still useful ? I feel it was meant for the demo VoiceRecog

Reply:
je suis quasi sur c'etait pour ca, je l'ai supprimé et tout passe toujours nikel